README.md

```diff
@@ -1,3 +1,15 @@
+# ALNS with Mabwiser
+
+The mabwiser integration isn't fully documented yet, but you can get started by looking at the example in `examples/tsp_mab.py`. The only difference from the provided `examples/travelling_salesman_problem.ipynb` is the following lines
+```
+# USE A MAB ALGORITHM AS A SELECTOR
```

Comment:
We need to make this usage example a bit more concrete. We can only focus on the non-contextual version here in the Readme and assume the standard ALNS parts, like operators, but flash the connection between MAB creation and ALNS calling. 

Ideally, it would be great to add a notebook under /examples, say mabwiser_alns.py that has at least two things:
1) the full example from readme that actually runs and compiles
2) a contextual version (does not have to super meaningful)